tests/test_never_type/test_runtime.py

```diff
@@ -0,0 +1,30 @@
+from pytest import mark, raises
+
+from basedtyping import Never, issubform
+
+
+def test_isinstance() -> None:
+    # https://github.com/python/mypy/issues/11903
+    assert not isinstance(  # type:ignore[misc]
+        1,
+        # https://github.com/KotlinIsland/basedmypy/issues/136
+        Never,  # type:ignore[arg-type]
+    )
+
+
+# https://github.com/python/mypy/issues/11903 and/or https://github.com/KotlinIsland/basedmypy/issues/24
+@mark.xfail
+def test_issubform_true() -> None:
+    """https://github.com/KotlinIsland/basedtyping/issues/22"""
+    assert issubform(Never, int)  # type:ignore[arg-type]
```

Comment:
So annotating issubform with `_SpecialForm` didn't fix this?

Reply:
nope but changing it to `typing.NoReturn` did, i was using `mypy_extensions.NoReturn` when it was a type alias to get around the type alias issue